agoraStreaming/src/Screens/Auth/index.tsx

```diff
@@ -0,0 +1,19 @@
+import {GoogleSigninButton} from '@react-native-google-signin/google-signin';
+import React, {FC} from 'react';
```

Comment:
please move 2nd line on the first place, cause it is React and will be better to write it firstt